sale_coupon_promotion_generate_coupon/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Generate coupons in another coupon program in sales",
+    "summary": "Allows to generate pending coupons in a coupon program in sales",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Sales Management",
+    "website": "https://github.com/OCA/sale-promotion",
+    "depends": ["coupon_promotion_generate_coupon"],
```

Comment:
You can't remove the `sale_coupon` dependency
```suggestion
    "depends": ["sale_coupon", "coupon_promotion_generate_coupon"],
```

Reply:
thanks :)